rbac/management/audit_log/model.py

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Model for audit logging."""
+from django.db import models
+
+
+class AuditLog(models.Model):
+    """An audit log."""
+
+    GROUP = "group"
+    ROLE = "role"
+    USER = "user"
+    PERMISSION = "permission"
+    RESOURCE_CHOICES = (
+        (GROUP, "Group"),
+        (ROLE, "Role"),
+        (USER, "User"),
+        (PERMISSION, "Permission"),
+    )
+
+    DELETE = "delete"
+    ADD = "add"
+    EDIT = "edit"
+    CREATE = "create"
+    REMOVE = "remove"
+    ACTION_CHOICES = (
+        (DELETE, "Delete"),
+        (ADD, "Add"),
+        (EDIT, "Edit"),
+        (CREATE, "Create"),
+        (REMOVE, "Remove"),
+    )
+
+    date = models.DateField(auto_now_add=True)
```

Comment:
I am not sure if I understand correctly, what value we want to have in? only date when the record is created? 
so maybe we can think about naming, in other RBAC models we are using `created` for this purpose
🤔 but I am just thinking loud
`created = models.DateField(auto_now_add=True)`

Reply:
Maybe date_created? 

For this value, date is pointing to the day/month/year a person makes any changes and is then logged into the audit log database! 
which is why I was thinking of just "date" it's just recording the date the changes were made, but I think date_created for that specific audit log also makes sense as well! 
What do you think?

Reply:
I don't have much experience in this field but do we really don't want to have time too? in the ticket there is `timestamp`requirement under ACs 🤔 and I like `date_created` or `created`

Reply:
Sounds good!! I'll change it then to date_created! 

I was basing off the fields off of the layout that's within this ticket: 
https://issues.redhat.com/browse/RHCLOUD-26581  


But I can also try to add the time!! I'll just have to figure out how to do that XDDD

Reply:
awesome, thank you